boms/legacy-expansion/pom.xml

```diff
@@ -68,135 +68,7 @@
-                        <groupId>javax.enterprise</groupId>
-                        <artifactId>cdi-api</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
-
-            <dependency>
-                <groupId>io.smallrye.config</groupId>
-                <artifactId>smallrye-config-common</artifactId>
-                <version>${legacy.version.io.smallrye.smallrye-config}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>io.smallrye.config</groupId>
-                <artifactId>smallrye-config-core</artifactId>
-                <version>${legacy.version.io.smallrye.smallrye-config}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>io.smallrye.config</groupId>
-                <artifactId>smallrye-config-source-file-system</artifactId>
-                <version>${legacy.version.io.smallrye.smallrye-config}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.eclipse.microprofile.config</groupId>
-                <artifactId>microprofile-config-api</artifactId>
-                <version>${legacy.version.org.eclipse.microprofile.config.api}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.eclipse.microprofile.metrics</groupId>
-                <artifactId>microprofile-metrics-api</artifactId>
-                <version>${legacy.version.org.eclipse.microprofile.metrics.api}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.eclipse.microprofile.openapi</groupId>
-                <artifactId>microprofile-openapi-api</artifactId>
-                <version>${legacy.version.org.eclipse.microprofile.openapi}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.eclipse.microprofile.opentracing</groupId>
-                <artifactId>microprofile-opentracing-api</artifactId>
-                <version>${legacy.version.org.eclipse.microprofile.opentracing}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>org.apache.geronimo.specs</groupId>
-                        <artifactId>*</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
-
-            <dependency>
-                <groupId>org.eclipse.microprofile.rest.client</groupId>
-                <artifactId>microprofile-rest-client-api</artifactId>
-                <version>${legacy.version.org.eclipse.microprofile.rest.client.api}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>*</groupId>
-                        <artifactId>*</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
+            <!-- Dependencies specific to this bom. Keep sorted by maven GA -->
```

Comment:
After this PR, legacy-ee and legacy-expansion are not adding any specific dependencies, so, they could be removed. I'm not asking to remove them, they could be there for future usage and I think this is intentional.

Reply:
Yes, leaving these in place is intentional.